ietf/utils/test_runner.py

```diff
@@ -661,12 +712,63 @@ def setup_test_environment(self, **kwargs):
             s[1] = tuple(s[1])      # random.setstate() won't accept a list in lieu of a tuple
         factory.random.set_random_state(s)
 
+        if settings.validate_html is False:
+            print("     Not validating any generated HTML; "
+                  "please do so at least once before committing changes")
+        else:
+            print("     Validating all HTML generated during the tests")
+            self.batches = {"doc": [], "frag": []}
+
+            # keep the html-validate configs here, so they can be kept in sync easily
+            config = {}
+            config["frag"] = {
```

Comment:
Dunno, good question. @rjsparks?

Reply:
I would rather have test_runner settings in test_runner than in settings.py. These aren't settings production needs, and this seems to be the right level of encapsulation for them. But instead of trying to alter the object that comes from django.config.settings perhaps there should be a module object for them instead?